rpc/core/src/api/rpc.rs

```diff
@@ -290,8 +296,8 @@ pub trait RpcApi: Sync + Send + AnySync {
     }
     async fn get_coin_supply_call(&self, request: GetCoinSupplyRequest) -> RpcResult<GetCoinSupplyResponse>;
 
-    async fn get_daa_score_timestamp_estimate(&self, daa_scores: Vec<u64>) -> RpcResult<GetDaaScoreTimestampEstimateResponse> {
-        self.get_daa_score_timestamp_estimate_call(GetDaaScoreTimestampEstimateRequest { daa_scores }).await
+    async fn get_daa_score_timestamp_estimate(&self, daa_scores: Vec<u64>) -> RpcResult<Vec<u64>> {
```

Comment:
Why make this different from all the other response patterns for every other RPC call in here?

Reply:
There are 2 classes of functions.  Trait functions (all ending with `_call`) and implementation/helper/convenience wrappers.  `_call` functions use `Request` and `Response` structures.  Helper functions are meant to (where possible) eliminate that so that they look as close to "native" functions as possible, returning requested data types.

The only reason some helper functions return `Response` structs is because `Response` contains different data that is meant to be returned as a single struct/payload.  However, if you take a look at the following example (`get_block()`) https://github.com/aspectron/rusty-kaspa/blob/wallet-refactoring-q4-2023/rpc/core/src/api/rpc.rs#L136 you can see how it unwraps the payload. ...you just don't see it in many places because most functions do return a single response payload.

I also want to point out that the new `WalletApi` introduced in this PR follows the same design pattern  https://github.com/aspectron/rusty-kaspa/blob/wallet-refactoring-q4-2023/wallet/core/src/api/traits.rs#L189-L206